doc/website/LICENSE

```diff
@@ -0,0 +1,18 @@
+Couscous
```

Comment:
Copied until https://github.com/CouscousPHP/Template-Dark/pull/5 is merged. Then we will remove custom theme.